src/views/Home.vue

```diff
@@ -98,19 +109,83 @@ export default {
             } 
             
             return auxApoio
-        }
+        },
+
+        listarPostagemPorCategoria(categoria){
+            if(categoria == 'Todas'){
+                PostagemService.listarPostagem().then(Response => {
+                    this.postagemData = Response.data.posts;
+                })
+            }else{
+                PostagemService.listarPorCategoria(categoria).then(Response => {
+                    this.postagemData = Response.data.posts;
+                })
+            }
+        },
+
+        select(categoria){
+            this.listarPostagemPorCategoria(categoria)
+        },
+
+        focarTodas() {
+            document.getElementById("Todas").focus()
+        },
+    },
+
+    mounted(){
+        this.focarTodas()
+        this.listarPostagemPorCategoria('Todas') //method1 will execute at pageload
```

Comment:
Ao mover a chamada 137 para o created, o mouted pode ficar somente com o this.focarTodas().